compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -235,6 +235,19 @@ enum ProcessorFeatures {
   CPU_FEATURE_MAX
 };
 
+#ifndef _WIN32
+__attribute__((visibility("hidden")))
+#endif
+struct __processor_model {
```

Comment:
I'm not sure, probably this naming was chosen when it wasn't supposed to pass a struct as an argument (even for static functions). Maybe we need to rename it? But I'm ok with the current version.